packages/taiko-client/cmd/flags/common.go

```diff
@@ -28,6 +28,12 @@ var (
 		Category: commonCategory,
 		EnvVars:  []string{"L1_WS"},
 	}
+	L1PrivateTxEndpoint = &cli.StringFlag{
+		Name:     "l1.privateTx",
```

Comment:
```suggestion
		Name:     "l1.privateTxWs",
```

Reply:
Both http and ws are supposed to be supported.

Reply:
But in flag description it says websocket endpoint?

Reply:
Let me fix the description!